programmers/두-개-뽑아서-더하기.js

```diff
@@ -0,0 +1,13 @@
+function solution(numbers) {
+    let numbersum = [];
+    for(let a = 0; a < numbers.length; a++) {
+        for(let b = 0; b < numbers.length; b++) {
```

Comment:
a, b 보단 i, j 를 사용하는것이 좋습니다.
```suggestion
    for(let i = 0; i < numbers.length; i++) {
        for(let j = 0; j < numbers.length; j++) {
```